storage-explorer-apphost/src/main/java/com/microsoft/aspire/springsample/StorageExplorerAppHost.java

```diff
@@ -21,11 +21,8 @@ public void configureApplication(DistributedApplication app) {
 //            .withExternalHttpEndpoints()
 //            .withReference(blobStorage);
 
-        app.addDockerFile("storage-explorer", "Dockerfile")
+        app.addDockerFile("storage-explorer", "../storage-explorer/Dockerfile", "../storage-explorer")
```

Comment:
Paths in the manifest are relative to the app-host project, so this `..`'ing takes care of ensuring the correct path to the dockerfile and the build context (`.` would be treated as the app host project folder).